include/deal.II/numerics/data_out_faces.h

```diff
@@ -112,31 +112,16 @@ namespace internal
  *
  * @ingroup output
  */
-template <int dim, typename DoFHandlerType = DoFHandler<dim>>
-class DataOutFaces : public DataOut_DoFData<DoFHandlerType,
-                                            DoFHandlerType::dimension - 1,
-                                            DoFHandlerType::dimension>
```

Comment:
Shouldn't line 118 be `patch_space_dim = DoFHandlerType::space_dimension` or am I mistaken?

Reply:
I would guess so but the argument is called `patch_space_dim`. Run the tests and see what happens!

Reply:
No additional tests tests that fail occurred. I'll add a separate commit with a proposed change of this template argument.

Reply:
Well, no tests failed, but I am still undecided if it's the right choice to change that line. As we are creating patches for output, the actual `patch_space_dim` should be equivalent to `DoFHandlerType::dimension` and not `DoFHandlerType::space_dimension`.

@bangerth -- Do you remember?

Reply:
Let's postpone the decision and discuss the matter in a different PR.